internal/receiver/databricksreceiver/paginator.go

```diff
@@ -0,0 +1,83 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package databricksreceiver
+
+import "fmt"
+
+// paginator is extracted from apiPaginator for swapping out in unit tests
+type paginator interface {
+	jobsList() (out []job, err error)
```

Comment:
since this is an outlier:
```suggestion
	jobs() (out []job, err error)
```

Reply:
Oh yeah -- I was just using the names of the api methods. Fixed.